operator/controllers/execution/suite_test.go

```diff
@@ -36,6 +37,17 @@ var testEnv *envtest.Environment
 var ctx context.Context
 var cancel context.CancelFunc
 
+type fakeClock struct {
+	timeToAdd time.Duration
+}
+
+var FakeTime = time.Date(2023, 1, 1, 15, 0, 0, 0, time.UTC)
+var FakeClock = &fakeClock{timeToAdd: 0}
+
+func (f *fakeClock) Now() time.Time             { return FakeTime.Add(f.timeToAdd) }
+func (f *fakeClock) TimeTravel(d time.Duration) { f.timeToAdd += d }
+func (f *fakeClock) Reset()                     { f.timeToAdd = 0 }
+
```

Comment:
Is this still used?

Reply:
Not any more, I did not know at the time which solution we're going to use, so i kept both. I'll remove the fakeClock :)